examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -156,7 +184,7 @@ export class TaskList extends DataObject implements ITaskList {
         if (this._draftData?.get(id) === undefined) {
             return;
         }
-        const newTask = new Task(id, nameSharedString, prioritySharedCell);
+        const newTask = new Task(id, nameSharedString, prioritySharedCell, undefined, undefined, undefined);
```

Comment:
Should this pull from savedData rather than using undefined?

Reply:
Oh good point.